README.md

```diff
@@ -94,6 +94,41 @@ The component which orchestrates other components: a "glue" component. Provides
 a way to start the bridge. This is the component most examples use. Has
 dependencies on most of the components listed above.
 
+## `Logging`
+This component exposes access to the bridges log reporter. To use, you should
+install the optional packages `winston`, `winston-daily-rotate-file` and
+`chalk` to get nice formatted log lines, otherwise it will default to the JS
```

Comment:
we might want to recommend versions here, like `winston@3` or whatever

Reply:
I kinda hoped it would be implied by the package.json, as I think you can ask npm to install optionals.

Reply:
I tried to find information on npm installing deps that nested in a project, but couldn't find anything. Duplicating the versions (even just the major versions) is fine here, I think.